profiles/seccomp/generate.go

```diff
@@ -0,0 +1,35 @@
+// +build ignore
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/docker/docker/profiles/seccomp"
+)
+
+// saves the default seccomp profile as a json file so people can use it as a
+// base for their own custom profiles
+func main() {
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	f := filepath.Join(wd, "default.json")
+
+	// get the default profile
+	p := seccomp.GetDefaultProfile()
+
+	// write the default profile to the file
+	b, err := json.MarshalIndent(p, "", "\t")
+	if err != nil {
+		panic(err)
+	}
+
+	if err := ioutil.WriteFile(f, b, 0755); err != nil {
```

Comment:
s/0755/0644/ ? that json file is not going to execute itself any time soon :smile_cat: